lib/index.js

```diff
@@ -365,6 +343,24 @@ function stringToBuffer(string) {
 }
 
 /**
+ *
+ * Converts an ArrayBuffer to a Buffer
+ *
+ * @api private
+ */
+
+function arrayBufferToBuffer(data) {
```

Comment:
I believe we should only transfer the part relative to the view.

But as far as I tried, ArrayBufferView can handle only a part of ArrayBuffer.
That means we don't need to use `byteOffset` to retrieve data from ArrayBufferView. Or am I missing something?